tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -405,7 +405,7 @@ public function testConnectionIsClosed()
     {
         $this->_conn->close();
 
-        $this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
+        //$this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
 
         $this->_conn->quoteIdentifier('Bug');
```

Comment:
I guess the only way to properly test this without side effects is by using reflection. @Ocramius what do you think? Any other idea?